Frontend/ui-customer/src/Helpers/Utils.js

```diff
@@ -171,6 +171,16 @@ export const decodeErrorCode = function (a) {
     }
   };
 
+  export const decodeErrorPayload = function (a) {
+    try {
+      var errorPayload = JSON.parse(a).errorPayload;
+      return  errorPayload;
+    } catch (e) {
+      console.log(e);
+      return " ";
+    }
+  };
+
```

Comment:
@AmritpalSingh6749 since decode payload, code and message act as same can you make it configurable by passing string you want to access value of ?

Reply:
make common function and use same everywhere